raco/language/myrialang.py

```diff
@@ -1018,6 +1041,18 @@ def fire(self, exp):
         return exp
 
 
+def check_partition_equality(op, representation):
+    """Check to see if the operator has the required hash partitioning.
+    @param op operator
+    @param representation list of columns hash partitioned by,
+                        in the unnamed perspective
+    @return true if the op has an equal hash partitioning to representation
+    """
+
+    p = set([expression.ensure_unnamed(attr) for attr in op.partitioning().hash_partitioned])
```

Comment:
Nitpick: you can leave out the square brackets in the set constructor, making this a generator expression instead of a list comprehension.